canvas_modules/harness/cypress/support/canvas/verification-cmds.js

```diff
@@ -492,6 +492,56 @@ Cypress.Commands.add("verifyDetachedLinkPathToTarget", (trgNodeName, trgPortId,
 		});
 });
 
+Cypress.Commands.add("verifyWysiwygCommentStyles", (expectedStyles) => {
+	cy.get(".d3-comment-text")
+		.should("exist")
+		.then(($el) => {
+			if (expectedStyles.fontFamily) {
+				cy.wrap($el)
+					.should("have.css", "font-family")
+					.and("include", expectedStyles.fontFamily);
+			}
+
+			if (expectedStyles.fontSize) {
+				cy.wrap($el).should("have.css", "font-size", expectedStyles.fontSize);
+			}
+
+			if (expectedStyles.fontWeight) {
+				cy.wrap($el).should("have.css", "font-weight", expectedStyles.fontWeight);
+			}
+
+			if (expectedStyles.fontStyle) {
+				cy.wrap($el).should("have.css", "font-style", expectedStyles.fontStyle);
+			}
+
+			if (expectedStyles.textDecoration) {
+				cy.wrap($el)
+					.should("have.css", "text-decoration")
+					.and("include", expectedStyles.textDecoration);
+			}
+
+			if (expectedStyles.color) {
+				cy.wrap($el).should("have.css", "color", expectedStyles.color);
+			}
+
+			if (expectedStyles.textAlign) {
+				cy.wrap($el).should("have.css", "text-align", expectedStyles.textAlign);
+			}
+
+			if (expectedStyles.verticalAlign) {
+				cy.wrap($el).should("have.css", "vertical-align", expectedStyles.verticalAlign);
+			}
+
+			if (expectedStyles.backgroundColor) {
+				cy.wrap($el).should("have.css", "background-color", expectedStyles.backgroundColor);
+			}
+
+			if (expectedStyles.outline) {
+				cy.wrap($el).should("have.css", "display", expectedStyles.outline);
+			}
```

Comment:
The outline styling is added on the  `<div>` with class `d3-comment-text-scroll` not to the div with class `d3-comment-text`. (Unfortunately, applying the style there does not yield the correct appearance so I had to split the styling across `<div>`s.)  

<img width="276" alt="image" src="https://github.com/user-attachments/assets/1bb5b107-2350-47c7-8faf-82090ffaeee1">

Actually, here is a little extra improvement you can make as part of this item. Find the place in the code where we allocate the `border-width` to show or hide the outline and instead of using `border-width` change it to use `border-style` (either 'none' or 'solid') instead. Then change your test accordingly.